### PR TITLE
add missing bsg_test_dram_channel.cpp

### PR DESCRIPTION
**Pull request by tommydcjung**

### CODE REVIEW

File: software/mk/Makefile.paths [hunk 1]
```diff
@@ -3,4 +3,4 @@ BSG_IP_CORES_DIR ?= $(BSG_MANYCORE_DIR)/../basejump_stl
 BERKELEY_HARDFLOAT_DIR ?= $(BSG_MANYCORE_DIR)/../berkeley-hardfloat
 RISCV_BIN_DIR ?= $(BSG_MANYCORE_DIR)/software/riscv-tools/riscv-install/bin
 CAD_DIR ?= $(BSG_MANYCORE_DIR)/../bsg_cadenv
-BSG_MACHINE_PATH ?= $(BSG_MANYCORE_DIR)/machines/4x4
+BSG_MACHINE_PATH ?= $(BSG_MANYCORE_DIR)/machines/16x8
```
Comment (mrutt92) [inline software/mk/Makefile.paths]: why is this changed?

Comment (mrutt92) [reply software/mk/Makefile.paths]: oh nm.

Comment (tommydcjung) [reply software/mk/Makefile.paths]: it should be changed back to 4x4
